lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -221,7 +221,7 @@ function hasFraction(args) {
 }
 
 function isFraction(node) {
```

Comment:
Oh yeah - I was thinking no since the functions in that class check for more specific types of fractions. I think it would make sense to move this one once it gets defined more than once :)